sdk/src/cameras/itof-camera/camera_itof.cpp

```diff
@@ -102,11 +101,6 @@ CameraItof::CameraItof(
         m_isOffline = true;
     }
 
-    // Additional controls
-    if (m_adsd3500Enabled) {
-        m_controls.emplace("imagerType", "1");
```

Comment:
Can we keep this? We can use it to check if we have adsd3100 or adsd3030.

Reply:
I thought about this. I don't know when will it be useful. But could be useful at some point. But instead of control, it would be nice to have a method that could tell the imager type and even an enum that describes all supported types (3100 and 3030).

Reply:
I have implemented a method for this.